docs/rst/developer_guide/data_guide.rst

```diff
@@ -83,10 +83,65 @@ Modifying the data
 Each store requires a different protocol.
 
 ``small_files``
-    Install `pre-commit <https://pre-commit.com/>`_ and activate the hooks, this
-    will update the registry automatically upon commit. The rules used to create
-    the registry file are defined in the ``"registry_rules.yml"`` file. If, for
-    some reason, you cannot use pre-commit, then you must be very careful and
+    Install `pre-commit <https://pre-commit.com/>`_:
+    
+    .. code:: bash
+
+       pip install pre-commit
```

Comment:
These installation instructions are just a subset of what the pre-commit website suggests/recommends. I'd rather have here a link to the pre-commit website (telling people how to install third party software isn't really our job IMO).

Reply:
That's right, I agree